.github/workflows/ci.yml

```diff
@@ -115,21 +115,12 @@ jobs:
         with:
           python-version: "3.10"
 
-      - run: python -m pip install tox
-
       - name: Install virtualenv
-        run: git clone https://github.com/pypa/virtualenv.git
+        run: pip install virtualenv
         shell: bash
 
       - name: Test Nushell in virtualenv
-        run: |
-          cd virtualenv
-          # if we encounter problems with bleeding edge tests pin to the latest tag
-          # git checkout $(git describe --tags | cut -d - -f 1)
-          # We need to disable failing on coverage levels.
-          nu -c "open pyproject.toml | upsert tool.coverage.report.fail_under 1 | save patchproject.toml"
-          mv patchproject.toml pyproject.toml
-          tox -e ${{ matrix.py }} -- -k nushell
```

Comment:
i do not see `tox` tests in the new `test_virtualenv.nu`, is that normal?

Reply:
tox is Python test runner used in virtualenv repo to run the tests. We don't need it.

Reply:
ooh okok :+1: